src/quick-lint-js/diag/diagnostic-types-2.h

```diff
@@ -759,6 +759,13 @@ struct Diag_Duplicated_Cases_In_Switch_Statement {
   Source_Code_Span duplicated_switch_case;
 };
 
+struct Diag_Explicit_Fallthrough_Comment_In_Switch {
```

Comment:
Nit: We normally name the diagnostic struct after the *problem*, not the solution. Therefore this should be named something like `Diag_Fallthrough_Without_Comment_In_Switch`.